lib/screens/packs/packs_list_appbar.dart

```diff
@@ -17,6 +19,21 @@ class PacksListAppBar extends StatelessWidget {
   final Function packsListNav;
   final int currentIndex;
 
+  @override
+  _PacksListAppBarState createState() => _PacksListAppBarState();
+}
+
+class _PacksListAppBarState extends State<PacksListAppBar> {
+  @override
+  void didChangeDependencies() {
+    super.didChangeDependencies();
+    final repo = Provider.of<OnBoardingRepo>(context);
+    if (repo.showPackTutorial) {
+      showTutorial();
+      repo.setViewed(HiveKeys.kShowPackTutorial, false);
+    }
+  }
```

Comment:
Just an idea, but maybe...

```
OnBoardingRepo.of(context).maybeShow(Tutorial.showPack, showTutorial);
```

```
void maybeShow(Tutorial type, VoidCallback onShow){
if (showPackTutorial){
onShow();
setViewed(...)
}
}
```

Reply:
And then replace it everywhere where it's used

Reply:
Good idea, I’ll update